exchanges/exchanges.go

```diff
@@ -232,6 +238,23 @@ func (sticks Candlesticks) needsUpdate(bin candlestickKey) bool {
 	return time.Now().After(lastStick.Start.Add(bin.duration() * 2))
 }
 
+// UTILITY FUNCTIONS
+
+// Quickly print the thing to a JSON-encoded string.
+func jsonify(thing interface{}) string {
+	s, _ := json.MarshalIndent(thing, "", "    ")
+	return fmt.Sprint(string(s))
```

Comment:
What's the Sprintf for?

Reply:
I heard you like strings with your strings, so I made your string into a string. :-P